### ISSUE TITLE
pass command name to commands in args[0]

### ISSUE THREAD

**Issue by danyisill** [Demonstrandum/Simp-O-Matic#8]
I mean, that's how it works in C.

**Comment by Demonstrandum** [comment]
That means I'll have to add one to every index I used in the whole program

**Comment by Demonstrandum** [comment]
You already know what the command name is based on the filename too innit

**Comment by danyisill** [comment]
Yeah but .alias should have alias name in args[0] and not the actual command.

**Comment by Demonstrandum** [comment]
arg[0] does have the alias name in it when calling !alias, or are you talking about the expansion of the alias?
